src/sst/elements/osseous/examples/vecshiftreg/AXI_port.h

```diff
@@ -0,0 +1,979 @@
+#ifndef DMATOP_H_
```

Comment:
@shubhamn94 There's nothing in here that infringes on anyone's copyright?

Reply:
It is an essent C-model generated from open sourced AntMicro's fastvdma chisel model. Here's the licence file: https://github.com/antmicro/fastvdma/blob/master/LICENSE

I see permission for commercial use there and some of the following content

   2. Grant of Copyright License. Subject to the terms and conditions of
      this License, each Contributor hereby grants to You a perpetual,
      worldwide, non-exclusive, no-charge, royalty-free, irrevocable
      copyright license to reproduce, prepare Derivative Works of,
      publicly display, publicly perform, sublicense, and distribute the
      Work and such Derivative Works in Source or Object form.

Reply:
@gvoskuilen @feldergast @nmhamster I'm a little worried about including this file. Any comments?

Reply:
Do we want to create a separate repo for examples (that may allow us to get around any copyright issues)? Could we host examples under SST or would we need a different project?

Reply:
@hughes-c 
I would like to mention that the current example does not require AXI support. So, I can turn that file down in next commit if required.

Reply:
Even under a separate repo we should check with copyright.

Reply:
Osseous examples pulled out in 84416d6